website/events/templates/events/admin/registrations_table.html

```diff
@@ -68,6 +69,7 @@
                 {% if cancellations %}
                 <td data-sortval="{{ registration.date_cancelled|date:'c' }}">{{ registration.date_cancelled }}</td>
                 {% endif %}
+                <td data-sortval="{{ registration.remarks }}">{% if registration.remarks %}{{ registration.remarks }}{% endif %}</td>
```

Comment:
I think you can use a filter that automatically sets the output to '-' or something

Reply:
https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#default-if-none

That doesn't require the if statement